tests/integration/src/tests/concurrent.rs

```diff
@@ -0,0 +1,101 @@
+#![allow(dead_code)]
+use fluvio::{Fluvio, Offset};
+use futures_lite::StreamExt;
+use fluvio_future::task::spawn;
+use md5::Digest;
+use std::sync::mpsc::{Receiver, Sender};
+use fluvio_system_util::bin::get_fluvio;
+use fluvio_command::CommandExt;
+use crate::TestOption;
+
+const TOPIC: &str = "test-bug";
+const PARTITION: i32 = 0;
+
+type Record = Vec<u8>;
+
+#[fluvio_test_macro::fluvio_test(topic = "concurrent")]
+fn concurrent_test() {
+    // Hardcoding for now, until ToTrait is implemented for TestOption
+    let test_option = TestOption::default();
```

Comment:
Note that the only thing that test_option is used for in this particular test case is to manually set up the topic. If the macro has already covered the topic setup, then TestOption is not needed here.

Reply:
Good to know. Though, it would be convenient to have the option to inject the number of iterations. Especially if we want to scale or longevity test on a regular frequency.